mod/Domestication/common/special_projects/00_a_hdgoverride_projects_horizon_signal.txt

```diff
@@ -0,0 +1,58 @@
+# In its own special file because this follows FIOS override rules
```

Comment:
We can probably just remove the modified horizon signal stuff altogether.  I started working on this a while back, but never really got anywhere with it.